src/completion-provider.js

```diff
@@ -127,10 +128,15 @@ class CompletionProvider {
     const webpackConfig = this.fetchWebpackConfig(projectPath);
 
     let moduleSearchPaths = get(webpackConfig, 'resolve.modulesDirectories', []);
+
     moduleSearchPaths = moduleSearchPaths.filter(
       (item) => vendors.indexOf(item) === -1
     );
 
+    if (moduleSearchPaths.length === 0) {
```

Comment:
Is this necessary? I thought maybe it would improve performance marginally but now that I review it this seems redundant since it would eventually just evaluate as `Promise.all([])`. Am I correct that this can be removed?